user-guide/Advanced_Modules/SRM/SRM_Quarantine.md

```diff
@@ -0,0 +1,271 @@
+---
+uid: SRM_Quarantine
+---
+
+# Quarantine
+
+## What is the quarantine state?
+
+The quarantine state is a special state of a booking, that indicates the booking cannot start because one or more resources that were assigned are no longer available. Bookings are automatically moved to this state when the ResourceManager detects such a situation. Usually this is because overlapping bookings using the same resource are added or modified and the resource does not have enough capacity or concurrency to support all bookings.
+
+Bookings that are in a quarantine state will have the ``IsQuarantined`` flag set to true, and will have the status set to ``Pending``. These bookings will still reserve the non-quarantined resources, the same way they would if they were in a regular ``Pending`` state. The resources that are no longer available are still present on the booking in the ``QuarantinedResources`` collection, along with some information about why these resources were moved to quarantine.
+
+## Practical example
+
+Consider a system that has two existing bookings in the confirmed state, each using a resource *Resource one*. *Resource one* has a capacity defined, with a maximum bookable value of 10.
+
+![Quarantine example before](~/user-guide/images/quarantine_example_existing.png)
+
+If a third booking is added as configured below, there would not be enough capacity on *Resource one* to support this booking. If the addition of *Booking three* is forced anyway, the ResourceManager would resolve the conflict as follows: 
+
+![Quarantine example result](~/user-guide/images/quarantine_example_result.png)
+
+In the diagram above *Booking two* was moved to quarantine, because it has the lowest priority (see [Quarantine priority](#quarantine-priority)). *Resource one* is still partially assigned to *Booking two*, only the capacity necessary to resolve the conflict has been moved to quarantine. Since *Resource one* has a capacity of 10, only 1 capacity needed to be moved.
```

Comment:
Maybe repeat that the ``IsQuarantined`` flag will be set to true for Booking two and that both the normal and ``QuarantinedResources`` collection will contain a usage for that resource.